packages/wow-ui/src/components/Box/index.tsx

```diff
@@ -90,7 +109,7 @@ const Box = <T extends BoxVariantType = "text">({
         <Flex direction="column" gap="xxs">
           <styled.span
             color={textColor ? textColor : "textBlack"}
-            textStyle="h3"
+            textStyle={typeof text === "string" ? "h3" : undefined}
```

Comment:
p4;
text 가 string 일 때만 textStyle prop 을 주는 건 어떨까요?
```suggestion
           {...typeof text === "string" && {textStyle="h3"}}
```